presto-main/src/main/java/com/facebook/presto/server/QueryStateInfo.java

```diff
@@ -42,6 +42,7 @@
     private final String user;
     private final Optional<String> catalog;
     private final Optional<String> schema;
+    private final Optional<String> clientInfo;
```

Comment:
Nit: let's move this to be after user, since that matches the ordering in other places such as `ClientSession`